client/src/app/vets/components/vet-services/vet-services.component.html

```diff
@@ -0,0 +1,65 @@
+<div class="main_services">
+  <h2>Наші послуги</h2>
+</div>
+<section class="services-section">
```

Comment:
Why don't you load the list of services dynamically from an array?

Reply:
It wasn`t my task, it is an independent task that other team member was assigned